test/config.test1.yaml

```diff
@@ -50,19 +51,23 @@ build_shape_options:
   out_logging: true    # When true, logging is printed to console
   year: 2020    # reference year used to derive shapes, info on population and info on GDP
   nprocesses: 5    # number of processes to be used in build_shapes
+  worldpop_api: false
```

Comment:
may we call if world_pop_method and having as values "api" or "default"?
What do you think?